src/types.js

```diff
@@ -1,15 +1,80 @@
 // @flow
 
 export type TXOBJ = {
-  locktime: number,
+  nLockTime: number,
+  version: number, 
+  inputs: {
+    prevTxId: string,
+    sequenceNumber: string,
+    outputIndex: number,
+    script: string, // check
+    prevScriptPubKey: string, // check
+  }[],
+  outputs: { 
+    script: string,
+    satoshis: number 
+  }[],
```

Comment:
It looks like `locktime`, `ins` and `outs` are now `nLockTime`, `inputs`, and `outputs`.
That applies to "regular" transasctions too, right (`version = 1` I think)?